UPGRADE.md

```diff
@@ -34,3 +34,23 @@ Client queries will have to be changed like this:
 
 If you absolutely cannot break your clients, you can re-implement `@orderBy` in your
 project - it is a relatively simple `ArgManipulator` directive.
+
+### `@model` directive was repurposed 
+
+`@model` directive in v5 does what `@modelClass` did in v4. To implement similar functionality of `@model` from v4 use `@node` directive instead. 
```

Comment:
I think this part is somewhat confusing. How about we just give the minimal information needed to upgrade and link https://github.com/nuwave/lighthouse/pull/974 to give background information.

The upgrade guide should mention concrete steps, those are to 1. rename `@model` to `@node`  and 2. rename `@modelClass` -> `@model` (in that order).

Reply:
Would a comment like "Please apply changes in your project step by step from up to down" be enough? Alternatively we can define number to steps, and they then have to be applied in the incremental order.

Reply:
I tried a bit different approach, what do you think?

Reply:
Oh yes, that is much better!